ocs_ci/ocs/ocp.py

```diff
@@ -107,6 +107,28 @@ def get(
             command += " -o yaml"
         return self.exec_oc_cmd(command)
 
+    def describe(self, resource_name='', selector=None, all_namespaces=False):
+        """
+        Get command - 'oc describe <resource>'
+
+        Args:
+            resource_name (str): The resource name to fetch
+            selector (str): The label selector to look for
+            all_namespaces (bool): Equal to oc describe <resource> -A
+
+        Example:
+            describe('my-pv1')
+
+        Returns:
+            dict: Dictionary represents a returned yaml file
+        """
+        command = f"describe {self.kind} {resource_name}"
+        if all_namespaces and not self.namespace:
+            command += "-A"
```

Comment:
I think you need to add space " -A" instead of "-A"

Reply:
Should I edit here as well https://github.com/red-hat-storage/ocs-ci/pull/368/files/fa17b3cd052a374b19c72d61adf906a9380c5d91#diff-0acc23436a82754145dad2f2c3e0536dL103

Reply:
Done